src/install_maxtext_extra_deps/install_github_deps.py

```diff
@@ -1,91 +0,0 @@
-# Copyright 2025 Google LLC
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#    https://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-"""Installs extra dependencies from a requirements file using uv.
-
-This script is designed to be run to install dependencies specified in
-'extra_deps_from_github.txt', which is expected to be in the same directory.
-It first ensures 'uv' is installed and then uses it to install the packages
-listed in the requirements file.
-"""
-
-import subprocess
-import sys
-from pathlib import Path
```

Comment:
Why is this file deleted? Don't we still need this?

Reply:
Yeah but the Python file doesn't do anything that `pip install -r <txt_file>` doesn't. And that command is a regular part of peoples workflow, so no disadvantage in just making that the recommendation.

It's weird to have to execute a Python script that calls Popen on pip

Reply:
I'm not following why this is needed though. IMO it is easier for the customer to just run the existing workflow which is provided when they pip install maxtext. If we make this change, they will need to take an extra step to find the right tag and track down a longer URL for the command

Reply:
`git tag` is the release approach recommended by goreleaser, what I've been pushing for. `/blob/{PYPI_VERSION}` should be the scheme, as this is the most common approach.

Cleanest way of handling dependencies whence pypi restricts github deps.